debian.aws/config/OVERRIDES

```diff
@@ -0,0 +1,234 @@
+#
+# Disable various "net" modules which we don't use.
+#
+CONFIG_6LOWPAN=n
+CONFIG_6LOWPAN_NHC=n
+CONFIG_BT=n
+CONFIG_CAIF=n
+CONFIG_HAMRADIO=n
+CONFIG_IEEE802154=n
+CONFIG_NET_9P=n
+CONFIG_NET_DSA=n
+CONFIG_NFC=n
+CONFIG_RFKILL=n
+CONFIG_WIMAX=n
+CONFIG_WIRELESS=n
+
+#
+# Disable various "drivers" modules which we don't use.
+#
+CONFIG_ACCESSIBILITY=n
+CONFIG_ATA=n
+CONFIG_ATM_DRIVERS=n
+CONFIG_AUXDISPLAY=n
+CONFIG_BCMA=n
+CONFIG_CONNECTOR=n
+CONFIG_CRYPTO_HW=n
```

Comment:
I wonder whether this could inadvertently and silently hurt us. I'm no expert here, but is it possible that the presence of crypto hardware can be used to silently boost performance of crypto functions in a virtual machine, and that removing this support could disable this kind of capability?

Reply:
is there a way to test that? alternatively, I can just leave this out, and keep using the default.. we can always re-evaluate later.. I go ahead and do that, so we can move this forward.. are you OK with that? i.e. just remove this line..

Reply:
I'm not sure, it might be safer to leave `CONFIG_CRYPTO_HW` the way it was especially since it's security-related (unlikely to raise any security red flags).

Reply:
right, I agree.. I've removed `CONFIG_CRYPTO_HW` from the override file.